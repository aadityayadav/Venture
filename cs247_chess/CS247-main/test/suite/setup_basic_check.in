setup
+ K e1
+ k e8
+ P d7
done
- d7
done
