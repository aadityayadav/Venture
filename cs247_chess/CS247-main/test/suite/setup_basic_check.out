The initial board cannot have a player in check!
____k___
________
________
________
________
________
________
____K___
